test/com/google/enterprise/gsafeed/RecordTest.java

```diff
@@ -36,7 +43,7 @@ public void testRecord() throws Exception {
         "<record url='http://example.com'"
         + " displayurl='http://example.com' action='add'"
         + " mimetype='text/plain'"
-        + " last-modified='Tue, 6 Nov 2007 12:45:26 GMT'"
+        + " last-modified='Tue, 06 Nov 2007 12:45:26 +0000'"
```

Comment:
We might need to revisit this, because I seem to remember using GMT, and mangling the formatter output in connector manager. Yeah, see Value.java in the connector/spi package. RFC 822 calls for "GMT" rather than "+0000" as produced by Java.

Reply:
Was there a reason not to use "z" instead?